src/Routing/RouteBuilder.php

```diff
@@ -354,7 +354,7 @@ public function resources(string $name, $options = [], $callback = null): void
         }
         $options += [
             'connectOptions' => [],
-            'inflect' => 'underscore',
+            'inflect' => 'dasherize',
```

Comment:
I wonder if we should use class const for these things. Easy to make typos like `dasherise` etc by accident.

Reply:
If someone makes a typo then tests will fail :slightly_smiling_face:. Also I don't think this is going to change again in the foreseeable future.

Reply:
I am also arguing more from the type of usability from the user/dev.
Setting the routes up - there are no tests, but the result silently might be different.
Too many magic strings usually are a bad thing - at least it would give devs the option to opt-out of it if they want to ;)